govuk-connect.gemspec

```diff
@@ -7,7 +7,7 @@ require "govuk_connect/version"
 Gem::Specification.new do |s|
   s.name = "govuk-connect"
   s.version = GovukConnect::VERSION
-  s.required_ruby_version = ">= 2.3.0"
+  s.required_ruby_version = ">= 2.4.0"
```

Comment:
In light of this do we want to do matrix builds on multiple Ruby versions on CI?

Reply:
I've pushed a commit to try this, but it turns out we have a problem with conflicting gem versions. Normally one has one Gemfile for each Ruby version.

I'd like to suggest the minimum version is more of an 'indicator' than something we officially support. The extra testing and maintenance of the matrix doesn't seem worth it.